main.go

```diff
@@ -86,6 +86,16 @@ func main() {
 		log.Fatal(errors.AddContext(err, fmt.Sprintf("cannot connect to ClamAV on %s:%s", clamIP, clamPort)))
 	}
 
+	// Connect to Blocker.
+	scanner.BlockerIP = os.Getenv("BLOCKER_IP")
```

Comment:
Are you sure? I just removed all constants from accounts because it seemed to be confusing. We only use this value once. I don't mind having constants but it did seem like unnecessary overhead.

Reply:
Yeah, it's just easier to have a single place at the top of a file to see all environment variables which are in use with docstrings explaining what they are for and what their values are supposed to look like.

Reply:
OK, then I'll do the same for blocker and accounts as well. Better be consistent.

F/U: https://github.com/SkynetLabs/malware-scanner/issues/10